cmd/detect.go

```diff
@@ -75,6 +75,16 @@ func runDetect(cmd *cobra.Command, args []string) {
 		detector.AddGitleaksIgnore(filepath.Join(source, ".gitleaksignore"))
 	}
 
+	// ignore findings from the baseline (an existing report in json format generated earlier)
+	baselinePath, _ := cmd.Flags().GetString("baseline-path")
+	if baselinePath != "" {
+		detector.AddBaseline(baselinePath)
+		if err != nil {
+			log.Fatal().Msgf("Could not load baseline. The path must point of a gitleaks report generated using the default format: %s", err)
+		}
+		err = nil
```

Comment:
I don't think this is necessary, `log.Fatal()` will exit the program

Reply:
This was actually to meet andrews point:

Reply:
`log.Fatal()` will exit the program immediately after logging a message

Reply:
Using log.Error()  instead